### PR TITLE
Update inception quantized model path

### PR DESCRIPTION
**Pull request by hx89**
Update inception quantized model path. Also remove the auto_logits from model before loading pretrained quantized model.

Tested by running:
python references/classification/train_quantization.py  --data-path='/mnt/fair/imagenet_full_size/' --device='cpu' --backend='fbgemm' --test-only --model='inception_v3'

Got accuracy similar to https://github.com/pytorch/vision/tree/master/references/classification#int8-models:
Acc@1 77.176 Acc@5 93.354

### CODE REVIEW

**Comment by hx89** [comment]
Reported numbers are updated in #1975.
